src/lib/libast/tests/string/base64.c

```diff
@@ -0,0 +1,61 @@
+#include "config_ast.h"  // IWYU pragma: keep
+
+#include <stdint.h>
+#include <string.h>
+
+#include "ast.h"
+#include "terror.h"
+
+struct base64_tests {
+    const char *string;
+    const char *base64_string;
+};
+
+struct base64_tests tests[] = {
+    { "VIJZoMLCe1dwwqMiPKC1", "VklKWm9NTENlMWR3d3FNaVBLQzE=" },
+    { "CHCqOznM5peGMaIoccsL", "Q0hDcU96bk01cGVHTWFJb2Njc0w=" },
+    { "RbaVtZYLF9FmuOemXKzi", "UmJhVnRaWUxGOUZtdU9lbVhLemk=" },
+    { "2qKoXd2yH3dcoywNQfDL", "MnFLb1hkMnlIM2Rjb3l3TlFmREw=" },
+    { "iVFjwRZTPrXzQoEbnisP", "aVZGandSWlRQclh6UW9FYm5pc1A=" },
+    { "JDkAUes8uamPqIPwQKGV", "SkRrQVVlczh1YW1QcUlQd1FLR1Y=" },
+    { "bf0bCcTT1LySrR9eHi53", "YmYwYkNjVFQxTHlTclI5ZUhpNTM=" },
+    { "wwsbeqm8xYinra82fDlx", "d3dzYmVxbTh4WWlucmE4MmZEbHg=" },
+    { "pFkDWTcwbGATCiPJpkhy", "cEZrRFdUY3diR0FUQ2lQSnBraHk=" },
+    { "ENwnLtgW5VLvNicthw0N", "RU53bkx0Z1c1Vkx2TmljdGh3ME4=" },
+    { NULL, NULL }
+};
+
+// TODO: What shall we do with the test in 'src/lib/libast/tests/misc/base64.c' ?
```

Comment:
Might as well eliminate it. Looking at it I see it only writes a count of the errors to stdout. The Meson test framework won't actually register a failure of that test even if it does fail.